src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -196,8 +196,8 @@ ZoneSet::ZonesFromPoint(POINT pt) const noexcept
     for (const auto& [zoneId, zone] : m_zones)
     {
         const RECT& zoneRect = zone->GetZoneRect();
-        if (zoneRect.left - m_config.SensitivityRadius <= pt.x && pt.x <= zoneRect.right + m_config.SensitivityRadius &&
-            zoneRect.top - m_config.SensitivityRadius <= pt.y && pt.y <= zoneRect.bottom + m_config.SensitivityRadius)
+        if (zoneRect.left /* - m_config.SensitivityRadius */ <= pt.x && pt.x <= zoneRect.right /* + m_config.SensitivityRadius  */&&
+            zoneRect.top /* - m_config.SensitivityRadius */ <= pt.y && pt.y <= zoneRect.bottom /* + m_config.SensitivityRadius */)
```

Comment:
If you need to make changes to this function, they need to be done only when the `ClosestCenter` algorithm is selected.